lib/components/normal-components/Chip.ts

```diff
@@ -38,11 +38,12 @@ export class Chip<PinLabels extends string = never> extends NormalComponent<
     const { db } = this.root!
     const { _parsedProps: props } = this
 
-    const pinCount =
+    const totalPinCount =
```

Comment:
this var name is actually misleading
```suggestion
    const pinCountFromSchArrangement =
```